Sim/doc/FastSimulationUsingPapas.md

```diff
@@ -0,0 +1,137 @@
+
+PAPAS propagates stable generated particles through a simple detector model.
+
+In the tracker, charged particles may be detected as tracks, taking into account the acceptance, efficiency, and momentum resolution of this detector.
+
+In the calorimeters, particles are detected as energy deposits. The energy deposits are modelled by taking into account the following detector properties: energy resolution, acceptance, energy thresholds and characteristic size. The latter is defined as the distance between two clusters below which the two clusters cannot be resolved and are considered as a single cluster.
+
+A particle flow algorithm then runs over the simulated tracks and clusters to identify and reconstruct charged hadrons, photons, and neutral hadrons. These particles can then be used in Heppy as an input to higher-level algorithms like jet clustering, or directly in the analysis.
+
+TIP Electrons and muons are passed through PAPAS without any modification, and the user is responsible for applying their own efficiency and resolution models. The hadronic decay products of tau leptons are simulated just like other hadrons and photons.
+
+A CMS-like detector model is provided as an example.
+
+
+Running Papas in FCCSW
+-------------------------------------------------
+
+The input is a file of fcc generated particles eg using Pythia see [tutorial](https://github.com/HEP-FCC/fcc-tutorials/blob/master/FccSoftwareGettingStartedFastSim.md#generate-events-with-pythia8)
+
+The Gaudi interface to Papas, PapasAlg provides following configurable set tools
+ * PapasImportParticlesTool - reads in gen particles
+ * PapasSimulatorTool  - simulates smeared clusters and tracks
+ * PapasMergeClustersTool - merges clusters
+ * PapasBuildBlocksTool - builds blocks of linked clusters and tracks
+ * PapasSimplifyBlocksTool - simplifies the blocks
+ * PapasPFReconstructorTool - reconstructs particles based on the blocks
+ * PapasExportParticlesTool - exports papas reconstructed particles to fcc particles
+
+Papas uses a Papas::Event to store the collections of particles, clusters, tracks, blocks that it produces. The Papas::Event is passed to each of the tools and new collections are then added into the papas::Event. Thus subsequent tools can access previously made collections.
+  Once a collection has been added to the event it can no longer be changed.
+
+ The papas::Event uses a short hand to label the collections that it contains:
+   the code is a two letter string eg "es"
+
+the first letter is the type of object contained in the collection
+```
```

Comment:
Please add an empty line between the text and the code block (the jekyll parser will get confused otherwise).